### PR TITLE
Adding openap

### PR DESCRIPTION
**Pull request by xoolive**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there => please post a comment @junzis
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by junzis** [comment]
Hi @conda-forge-linter and @xoolive, yes, I will be the maintainer.

**Comment by xoolive** [comment]
@conda-forge/help-python This is ready for review

File: recipes/openap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - scipy
+    - scikit-fuzzy
+    - pandas
+    - pyyaml
+    - matplotlib
```
Comment (mariusvniekerk) [inline recipes/openap/meta.yaml:27]: Can this use

```suggestion
    - matplotlib-base
```

**Comment by xoolive** [comment]
Thank you @mariusvniekerk, did my homework and you are right.

**Comment by xoolive** [comment]
@conda-forge/help-python Would it be possible to proceed with this package? It serves as a dependency to another recipe I need to prepare as well... :sweat_smile:

**Review by dopplershift — APPROVED** [review]

**Pull request merged**